tests/handler_linearity.tests

```diff
@@ -0,0 +1,42 @@
+Use linear variables in deep handlers (1)
```

Comment:
Excellent! The tests all seem to be "failing tests", i.e. they are meant to fail. It would be good to add some "success tests", i.e. tests that are meant to succeed.

Reply:
Not so easy to add success tests. I added one test about using linear vars in clauses of shallow handlers. Not sure how to write operations which use linear parameters 🤔